uwsdora.module

```diff
@@ -106,3 +106,52 @@ function uwsdora_views_api() {
     'path' => drupal_get_path('module', 'uwsdora') . '/views',
   );
 }
+
+/**
+ * Implements hook_convert_mods_to_citeproc_jsons_alter().
+ */
+function uwsdora_convert_mods_to_citeproc_jsons_alter(&$output, $mods) {
+  module_load_include('inc', 'citeproc', 'includes/converter');
+
+  $pages = $mods->xpath("//mods:mods[1]/mods:relatedItem[@type='host']/mods:part/mods:extent[@unit='pages']");
+
+  if (empty($pages)) {
+    $pages = $mods->xpath("//mods:mods[1]/mods:relatedItem[@type='host']/mods:part/mods:extent[@unit='page']");
+  }
+
+  $pages = (!empty($pages) ? $pages[0] : NULL);
```

Comment:
Could... just:
```php
$pages_element = reset($pages);
```

... Would also need to adjust the variable usage below. Not sure of the exact provenance of this code, but just noting: It's generally good to avoid variable reuse.